tests/unit/connect.c

```diff
@@ -49,16 +49,18 @@ int main(int argc, char *argv[])
 {
   (void)argc;
   (void)argv;
-
-  drizzle_options_st *opts = drizzle_options_create();
+  drizzle_options_st *opts;
```

Comment:
Could you explain in the commit message why you are making this change as its not obvious just looking at it.